sys/src/9/port/sysfile.c

```diff
@@ -868,6 +882,22 @@ write(int fd, void *p, int32_t n, int64_t off)
 	return r;
 }
 
+void
+syswrite(Ar0* ar0, ...)
+{
+	va_list list;
+	va_start(list, ar0);
+	int fd = va_arg(list, int);
+	void *buf = va_arg(list, void *);
+	long nbytes = va_arg(list, long);
+	int64_t offset = -1ULL;
```

Comment:
note that this is right, and the above is wrong, and you don't want to be inconsistent.

Reply:
because in original from NIX and NxM offset is not offp from func arg. I'm not sure why in harvey read and write were different in the sysfile.c imported in 2015.

Reply:
I don't understand